analyzer/metadata-plugin/src/main/java/eu/fasten/analyzer/metadataplugin/Main.java

```diff
@@ -59,7 +62,8 @@ public static void main(String[] args) {
     public void run() {
         var metadataPlugin = new MetadataDatabasePlugin.MetadataDBExtension();
         try {
-            metadataPlugin.setDBConnection(PostgresConnector.getDSLContext(dbUrl, dbUser));
+            metadataPlugin.setDBConnection(new HashMap<>(Map.of("java",
```

Comment:
I think it would be a bit more convenient to use `forge` as a key in the map of DSL Contexts instead of language. Because during runtime you choose the right context based on the forge you receive, not language. But if you prefer having a language there, I don't really have a problem with that.

Reply:
I didn't implement this piece of code, simply merged multiple-db-conc, before finalizing QA code.